### PR TITLE
add login/logout buttons to navbar

### PR DESCRIPTION
**Pull request by attwad**
image

After login:
image

Steps to try out locally:
```
# set ENV secret and client id from cloud project (I'm using a personal one for now until Gaurav gives us one)
./dev/run_stack.sh
Then go to http://localhost:3001/auth/google, accept the oauth prompt.
Then go to http://localhost:3002 to access the UI.
```

Because we run the server and the UI on different ports, clicking the login button doesn't work and you have to go to the localhost:3001/auth/google manually now , it will work in prod because they will be served from the same domain but until I find a better solution this manual step is needed.

Logout works as expected.

### CODE REVIEW

**Comment by attwad** [comment]
[Set up a login and auth mechanism.](https://trello.com/c/YKmdRRjW/97-set-up-a-login-and-auth-mechanism)

File: verification/curator-service/ui/src/components/EpidNavbar.tsx [hunk 1]
```diff
@@ -1,52 +1,84 @@
...
+                                data-testid="home-btn"
+                                edge="start"
+                                className={classes.menuButton}
+                                aria-label="menu">
+                                <HomeIcon />
+                            </IconButton>
+                        </Link>
+                        <Typography variant="h6" className={classes.title}>
+                            epid
                     </Typography>
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/EpidNavbar.tsx:68]: super-nit: increase indentation to match <Typography

**Comment by attwad** [comment]
No login isn't required anywhere yet. I'll add instructions once it's required.

File: verification/curator-service/ui/src/components/EpidNavbar.tsx (hunk 1 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/EpidNavbar.tsx:68]: thanks, weird I have autoformat on save but it didn't catch that. Done.

**Pull request merged**